content/influxdb/v1.8/about_the_project/releasenotes-changelog.md

```diff
@@ -9,32 +9,32 @@ menu:
 v2: /influxdb/v2.0/reference/release-notes/influxdb/
 ---
 
-## v1.8.4 [2020-01-27]
+## v1.8.4 [2020-02-01]
 
-### Bug fixes
-
-- Add durations to Flux logging, including the log compilation, execution, and total request duration. Previously, these stats were incorrectly logging `0.000ms`:
-
-  - `stat_total_duration`
-  - `stat_compile_duration`
-  - `stat_queue_duration`
-  - `stat_plan_duration`
-  - `stat_requeue_duration`
-  - `stat_execute_duration`
-
- Now, these durations are logged correctly. Note, `flux-enabled` and `flux-log-enabled` must be set to `true` in the [InfluxDB configuration file](/influxdb/v1.8/administration/config). For more information about InfluxDB logging, see [Log and trace with InfluxDB](/influxdb/v1.8/administration/logs).
-
-## v1.8.3 [2020-09-30]
+   > **Note:** InfluxDB 1.8.3 was not released. Features and bug fixes intended for 1.8.3 were rolled into InfluxDB 1.8.4.
 
 ### Features
 
-- Use latest version of InfluxQL package.
+- Use the latest version of InfluxQL package.
 - Add `-lponly` flag to [`influx export`](/influxdb/v2.0/reference/cli/influx/export/).
 - Add the ability to track number of values written via the [/debug/vars HTTP endpoint](/influxdb/v1.8/tools/api/#debug-vars-http-endpoint).
 - Update UUID library from [github.com/satori/go.uuid](https://github.com/satori/go.uuid) to [github.com/gofrs/uuid](https://github.com/gofrs/uuid).
+- Add `stat_total_allocated` to Flux logging.
+ To ensure Flux logging is enabled, set both `flux-enabled` and `flux-log-enabled` to `true` in the [InfluxDB configuration file](/influxdb/v1.8/administration/config). For more information about InfluxDB logging, see [Log and trace with InfluxDB](/influxdb/v1.8/administration/logs).
 
 ### Bug fixes
 
+- Add durations to Flux logging, including the log compilation, execution, and total request duration. Previously, the following stats were incorrectly logging `0.000ms`:
+
+  - `stat_total_duration`
+  - `stat_compile_duration`
+  - `stat_execute_duration`
+  - `stat_queue_duration`
```

Comment:
@kelseiv - we need to remove `stat_queue_duration`, `stat_plan_duration`, and `stat_requeue_duration` per Chris's earlier comment.  They are not valid in 1.x.  Correct, @wolffcm?

Reply:
Yes, that's right.